consensus/hotstuff/integration/integration_test.go

```diff
@@ -97,7 +98,10 @@ func TestThreeInstances(t *testing.T) {
 }
 
 func TestSevenInstances(t *testing.T) {
-	t.Skip()
+	if os.Getenv("TEST_FLAKY") == "" {
```

Comment:
Any thoughts on making a `SkipIf(t, "TEST_FLAKY", "message")` function?

Reply:
Sure, that's a good idea.